src/components/DashboardTable/DashboardTablePresentation.js

```diff
@@ -37,6 +37,7 @@ function DashboardTablePresentation({
                       {tableBody}
                     </tbody>
                   </table>
+                  {infiniteLoadBtn}
```

Comment:
Does`infiniteLoadBtn` being undefined cause errors on the other pages that use DashboardTable? Do we need a conditional or a default value for it for those pages?